evm/interchainTokenFactory.js

```diff
@@ -228,21 +238,30 @@ async function processCommand(_axelar, chain, chains, options) {
         }
 
         case 'deployRemoteCanonicalInterchainToken': {
-            const { tokenAddress, destinationChain, gasValue } = options;
+            const { tokenAddress, destinationChain, gasValue, env } = options;
+
+            const submittedGasValue = !gasValue
+                ? await calculateItsCrossChainGas({
+                      sourceChain: chain.axelarId,
+                      destinationChain,
+                      env,
+                      eventType: 'InterchainTokenDeployment',
+                  })
+                : gasValue;
 
             validateParameters({
                 isValidAddress: { tokenAddress },
                 isNonEmptyString: { destinationChain },
-                isValidNumber: { gasValue },
+                isValidNumber: { submittedGasValue },
             });
 
             validateChain(chains, destinationChain);
 
             const tx = await interchainTokenFactory['deployRemoteCanonicalInterchainToken(address,string,uint256)'](
                 tokenAddress,
                 destinationChain,
-                gasValue,
-                { value: scaleGasValue(chain, gasValue), ...gasOptions },
+                submittedGasValue,
+                { value: scaleGasValue(chain, submittedGasValue), ...gasOptions },
```

Comment:
did you look at what `scaleGasValue` does though? the estimated value should be scaled already

Reply:
I'd suggest making `estimateITSFee` accept `options.gasValue` as an arg, and have it check if it's `auto` to estimate it, or just scale it if it's a number, and fail otherwise